### ISSUE TITLE
XEC subbranch with Orf9b:R32L on Orf1a:A599T branch(471 seqs, 32 regions) and its recombinant with KP.3.1.1(135 sequences, 14 regions)

### ISSUE THREAD

**Issue by Over-There-Is** [cov-lineages/pango-designation#2813]
Transferred from sars-cov-2-variants/lineage-proposals#2173.
## XEC> G2060A(Orf1a:A599T)> G28378T(Orf9b:R32L)

Query: G2060A, G28378T, -A11983T,-C18687T, -G25234T,-C27944T
("-"s are added to remove convergent sequences)

Earliest sequence: 08-07 Schleswig-Holstein, Germany EPI_ISL_19364163
Latest sequence: 10-11 Asturias, Spain EPI_ISL_19497008
Regions detected: Denmark 9, UK 9, Germany 8, Italy 8, Netherlands 7, Sweden 7, France 5, Spain 4, US 3(1 from Denmark, 1 pooled, 1 not on GISAID), Luxembourg 2, Belgium 1

https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice1_genome_test_7f5f_7a0d10.json?c=country&label=id:node_7267532
![image](https://github.com/user-attachments/assets/23bcd1a9-6f0e-406a-896b-cedb547dd265)

<details><summary>Sequences:</summary>

EPI_ISL_19364163, EPI_ISL_19378139, EPI_ISL_19383090,
EPI_ISL_19410252, EPI_ISL_19415557, EPI_ISL_19417577,
EPI_ISL_19417585, EPI_ISL_19421034, EPI_ISL_19426850,
EPI_ISL_19431751, EPI_ISL_19433943, EPI_ISL_19435638-19435640,
EPI_ISL_19436766, EPI_ISL_19436780, EPI_ISL_19443858,
EPI_ISL_19444527, EPI_ISL_19447700, EPI_ISL_19447703,
EPI_ISL_19448892, EPI_ISL_19448913, EPI_ISL_19450798,
EPI_ISL_19451326, EPI_ISL_19451444, EPI_ISL_19451506,
EPI_ISL_19451675, EPI_ISL_19456847, EPI_ISL_19456850,
EPI_ISL_19456875, EPI_ISL_19458753, EPI_ISL_19463326,
EPI_ISL_19464961, EPI_ISL_19467664, EPI_ISL_19468204,
EPI_ISL_19468795, EPI_ISL_19472522, EPI_ISL_19477073,
EPI_ISL_19477087, EPI_ISL_19477236, EPI_ISL_19477238,
EPI_ISL_19477492, EPI_ISL_19477776, EPI_ISL_19478671,
EPI_ISL_19478700, EPI_ISL_19481383, EPI_ISL_19481474,
EPI_ISL_19481484, EPI_ISL_19481515, EPI_ISL_19481533,
EPI_ISL_19484080, EPI_ISL_19485103, EPI_ISL_19485195,
EPI_ISL_19485671, EPI_ISL_19486463, EPI_ISL_19488255,
EPI_ISL_19488579, EPI_ISL_19493850, EPI_ISL_19493952,
EPI_ISL_19494033, EPI_ISL_19497008, EPI_ISL_19497087,
PQ409725.1

</details>

## KP.3.1.1 / XEC

Query for recombinant: C1684A, C19716T (edited)

Mutations based on XEC:
C1684A, C7113T, C12616T / G28378T(Orf9b:R32L)
Breakpoint: 12616\~13121
C1684A is likely from KP.3.1.1> C1684A, C14120T, A14226G (2 seqs, DE 1, GB 1).

09-02~10-01, Bayern, Germany 3
Date unavailable, Ohio, US 1
Sequences:
EPI_ISL_19424919, EPI_ISL_19477344, EPI_ISL_19488853,
EPI_ISL_19497066

https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice1_genome_test_29260_7a80e0.json?c=country&label=id:node_7268393
![image](https://github.com/user-attachments/assets/1fa8377f-0adf-47ed-b4b0-7e210932da2c)

**Comment by FedeGueli** [comment]
I suggest to use this query: T3565C, G28378T, G2060A,-C18687T,-C27944T, -G25234T,-A11983T it finds 66

**Comment by Over-There-Is** [comment]
> I suggest to use this query: T3565C, G28378T, G2060A,-C18687T,-C27944T, -G25234T,-A11983T it finds 66

G2060A, G28378T, -A11983T,-C18687T, -G25234T,-C27944T is used now.

**Comment by Over-There-Is** [comment]
Ireland, Canada

**Comment by Over-There-Is** [comment]
Portugal via US, Austria, and recombinant +1(Denmark)

**Comment by FedeGueli** [comment]
109

**Comment by Over-There-Is** [comment]
Rec: Sweden

**Comment by Over-There-Is** [comment]
Australia, rec:Italy

**Comment by xz-keg** [comment]
@corneliusroemer also suggest to designate this and the recombinant here.

**Comment by Over-There-Is** [comment]
Israel, rec:Netherlands

**Comment by Over-There-Is** [comment]
Japan, rec:United Kingdom

**Comment by Over-There-Is** [comment]
Recombinant query edited: C1684A, C19716T

**Comment by FedeGueli** [comment]
> C1684A, C19716T

thx it doesnt seem slow at all

**Comment by xz-keg** [comment]
223/ 40 now.

**Comment by Over-There-Is** [comment]
Chile, rec:UK

**Comment by Over-There-Is** [comment]
Finland, Canary Islands

**Comment by Over-There-Is** [comment]
Rec:France

**Comment by Over-There-Is** [comment]
Mutations based on XEC:
C1684A, C7113T, C12616T / G28378T(Orf9b:R32L)
Breakpoint: 12616\~13121
C1684A is likely from KP.3.1.1> C1684A, C14120T, A14226G (2 seqs, DE 1, GB 1).

**Comment by FedeGueli** [comment]
> https://cov-spectrum.org/explore/World/AllSamples/Past6M/variants?nucMutations=G2060A%2CG28378T%2CA11983A%2CC18687C%2CG25234G%2CC27944C https://cov-spectrum.org/explore/World/AllSamples/Past6M/variants?nucMutations=C1684A%2CC19716T XEC+Orf9b:R32L is 25% faster than average and as fast as XEC*. The recombinant is 48% faster than average and 28% faster than XEC*.

yes it is fast. a bit less than that but still i dont see cluster effect so it may be real.

**Comment by FedeGueli** [comment]
ping  @corneliusroemer

**Comment by Over-There-Is** [comment]
New Zealand, Norway, Malaysia

**Comment by FedeGueli** [comment]
Ping @corneliusroemer this should be designated this non spike mutation is apparently beneficial

**Comment by Over-There-Is** [comment]
rec:Canada

**Comment by Over-There-Is** [comment]
Singapore

**Comment by Over-There-Is** [comment]
Brazil, rec:Australia

**Comment by Over-There-Is** [comment]
N>K679R CA-ON 9

**Comment by Over-There-Is** [comment]
S:S680Y by @xz-keg G2060A, C23601A, G28378T MY2, SG2, GB1
https://github.com/sars-cov-2-variants/lineage-proposals/issues/2088#issuecomment-2574301719

**Comment by FedeGueli** [comment]
ping @corneliusroemer this should be designated (lineage and recaombinant)

**Comment by xz-keg** [comment]
recombinant is now 111

**Comment by Over-There-Is** [comment]
New Caledonia(FR), Hong Kong(CN), Switzerland, rec:Switzerland, rec:Singapore, rec:Croatia

**Comment by Over-There-Is** [comment]
> S:S680Y by [@xz-keg](https://github.com/xz-keg) G2060A, C23601A, G28378T MY2, SG2, GB1 [sars-cov-2-variants/lineage-proposals#2088 (comment)](https://github.com/sars-cov-2-variants/lineage-proposals/issues/2088#issuecomment-2574301719)

https://github.com/cov-lineages/pango-designation/issues/2813#issuecomment-2578258073

**Comment by FedeGueli** [comment]
@corneliusroemer this should got designated.

**Comment by Over-There-Is** [comment]
Romania, China(Mainland, via US-NY), Argentina, Poland

**Comment by Over-There-Is** [comment]
XEC.18 & XEV

**Comment by FedeGueli** [comment]
> XEC.18 & XEV

via https://github.com/cov-lineages/pango-designation/commit/4a6e529c6a8f03328ddafd3800e1252708f75f71
